astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -435,3 +438,37 @@ def test_time_wcs(time_spectral_wcs_2d):
     # with a time axis.
 
     plt.subplot(projection=time_spectral_wcs_2d)
+
+
+@pytest.mark.skipif('MATPLOTLIB_LT_22')
+@pytest.mark.skipif('TEX_UNAVAILABLE')
```

Comment:
Why not combine them?
```suggestion
@pytest.mark.skipif('MATPLOTLIB_LT_22 or TEX_UNAVAILABLE')
```

Reply:
Done.